### PR TITLE
User story 18 - User Profile Info

### PR DESCRIPTION
**Pull request by MackHalliday**
- regular user profile show users info expect for password
- **updated show method in user controller to set @user = current_user**. This allows us to stub logins

- Rspec OK 
- SimpleCov 99.86% (Missing coverage for session controller method create, needs sad path test if visitor does not enter in correct information. Can create testing under user_story_13 branch later)

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ def create
   end
 
   def show
-    @user = User.find(session[:user_id])
+    @user = current_user
```
Comment (tschaffer1618) [inline app/controllers/users_controller.rb]: Nice use of a helper method! I could probably use that in the sessions controller as well.

File: spec/features/users/regular_users/show_spec.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  before :each do
+    @regular_user = User.create!(name: "George Jungle",
+                  address: "1 Jungle Way",
+                  city: "Jungleopolis",
+                  state: "FL",
+                  zipcode: "77652",
+                  email: "junglegeorge@email.com",
+                  password: "Tree123")
+
+    allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@regular_user)
```
Comment (tschaffer1618) [inline spec/features/users/regular_users/show_spec.rb]: I'm not great with stubs so maybe I'm just being dense, but I don't see how this is necessary for the test to pass. Why do we need a stub if we are creating an actual regular user?

Comment (MackHalliday) [reply spec/features/users/regular_users/show_spec.rb]: so basically, in the user#show method, if we need the ``@user =User.find(session[:user_id])`` cannot find the session[:user_id]

**Comment by MackHalliday** [comment]
Closing pull request to include user_story_19 since user_story_19 functionality depends on user_story_18

**Pull request closed without merging**